modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/view_commerce_checkout_delivery_group.jsp

```diff
@@ -0,0 +1,76 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+DeliveryGroupDisplayContext deliveryGroupDisplayContext = (DeliveryGroupDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+%>
+
+<div>
+	<div class="panel-body">
+		<div class="h5">
+			<liferay-ui:message key="shipping-address" />
+		</div>
+
+		<%
+		CommerceAddress commerceAddress = deliveryGroupDisplayContext.getCommerceAddress(request);
+		%>
+
+		<p>
+			<span><%= commerceAddress.getName() %></span>
+		</p>
+
+		<p>
+			<span><%= commerceAddress.getStreet1() %></span>
+		</p>
+
+		<c:if test="<%= Validator.isNotNull(commerceAddress.getStreet2()) %>">
+			<p>
+				<span><%= commerceAddress.getStreet2() %></span>
+			</p>
+		</c:if>
+
+		<c:if test="<%= Validator.isNotNull(commerceAddress.getStreet3()) %>">
+			<p>
+				<span><%= commerceAddress.getStreet3() %></span>
+			</p>
+		</c:if>
+
+		<%
+		Region region = commerceAddress.getRegion();
+		%>
+
+		<p>
+			<span><%= commerceAddress.getCity() %></span>
+			<span><%= StringPool.COMMA_AND_SPACE %></span>
+			<span><%= region.getName() %></span>
```

Comment:
I'm not sure but I think the region can remain null (and here we will have a NPE)

Reply:
also I think we need to call the getName(locale) like we do for the country